example_project/testApp/prototype.py

```
@@ -31,6 +32,52 @@
+        },
+        {
+            "name": "Training Project",
+            "description": "A project that is used to train employees or customers.",
+        },
+        {
+            "name": "Consulting Project",
+            "description": "A project that is used to consult customers or projects.",
+        },
+    ]
+
+    class Interface(ReadOnlyInterface):
```

Pylint (3.3.7)

[refactor] Too few public methods (0/2)

(R0903)